maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java

```diff
@@ -73,7 +73,6 @@
  * group, to expose them for use in successive builds in the chain.
  */
 public class MavenRepositorySession implements RepositorySession {
-
```

Comment:
Please add empty line back as there is always an empty line after the class line.